wolfcrypt/test/test.c

```diff
@@ -2125,18 +2125,27 @@ static wc_test_ret_t _SaveDerAndPem(const byte* der, int derSz,
     #if !defined(NO_FILESYSTEM) && !defined(NO_WRITE_TEMP_FILES)
         XFILE pemFile;
     #endif
+    #ifndef WOLFSSL_NO_MALLOC
         byte* pem;
+    #else
+        byte pem[FOURK_BUF];
```

Comment:
This may be a bit large for embedded targets.  Any chance you can trim this down to just the required size to pass the test?